lib/mods_display/fields/edition.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ModsDisplay
+  class Edition < Field
+    def fields
+      return_fields = @values.map do |value|
+        edition_value = Stanford::Mods::Imprint.new(value).send(:edition_vals_str)
+        next unless edition_value.present?
+
+        edition_value.gsub!(%r{ */$}, '')
```

Comment:
What is this doing and why?

Reply:
And should it be before the `edition_value.present?`?

Reply:
it's trimming spaces at the end of the string, if they are present.  Because these fields, as designed in MARC, are meant to be glommed together into a string, there can be trailing punctuation or spaces.   The regex is explicitly for spaces before the end of the string.